dogpile/cache/region.py

```diff
@@ -879,6 +882,7 @@ def get_value():
                     value.metadata["ct"]
                 )
             ):
+                log.info('Missing cache for key: %s', orig_key)
```

Comment:
wouldn't I want to log that the creator() function is running in gen_value() and how long it took to run ?